browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -39,6 +39,7 @@ export const style = scoped.css`
 
   .clock {
     font: ${font.large.semibold};
+    text-shadow: 0px 1px 2px rgba(0, 0, 0, 0.20);
```

Comment:
do we also need to add a text-shadow to the topSites tiles (as that text is also straight on the background)

Reply:
Good question. I had to look it up, but we already have a text shadow for that top sites text.